locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -1875,6 +1950,27 @@ msgstr "Nombre de fois utilisée"
 msgid "No active invites"
 msgstr "Aucune invitation active"
 
+#: bookwyrm/templates/settings/server_blocklist.html:6
+#, fuzzy
+#| msgid "Import Books"
+msgid "Import Blocklist"
+msgstr "Importer des livres"
+
+#: bookwyrm/templates/settings/server_blocklist.html:26
+#: bookwyrm/templates/snippets/goal_progress.html:5
+msgid "Success!"
+msgstr "Bravo !"
+
+#: bookwyrm/templates/settings/server_blocklist.html:30
+#, fuzzy
+#| msgid "Successfully imported"
+msgid "Successfully blocked:"
+msgstr "Importation réussie"
```

Comment:
Yes, this has been bugging me about how I've worded things. For English, I think "instance" is preferable, since that's what mastodon uses and it doesn't clash with the technical definition of a server